package NAlice.NWonderlogs;

import "mapreduce/yt/interface/protos/extension.proto";

message TUuidMessageId {
    optional string Uuid = 1 [json_name = "uuid", (NYT.key_column_name) = "uuid"];
    optional string MessageId = 2 [json_name = "message_id", (NYT.key_column_name) = "message_id"];
    optional uint64 TimestampMs = 3 [json_name = "timestamp_ms", (NYT.column_name) = "timestamp_ms"];
}
